openedx/core/djangoapps/organization_api/tests/test_views.py

```diff
@@ -0,0 +1,82 @@
+"""
+Tests for organization API.
+"""
+import json
+import unittest
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+from oauth2_provider.tests.factories import AccessTokenFactory, ClientFactory
+
+from student.tests.factories import UserFactory
+from util import organizations_helpers
+
+
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+class OrganizationsAPITests(TestCase):
```

Comment:
Agreed -- and I recommend that said method/test fails when it encounters unexpected attributes.